packages/cx-layout/src/components/Search/DesktopSearch/token.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { addProps, replaceWith } from '@bodiless/fclasses';
+import SearchIcon from '../assets/SearchIcon';
+import { asDesktopSearchToken } from './DesktopSearchClean';
+
+/**
+ * Token that defines a basic header.
+ */
+const Base = asDesktopSearchToken({
+  Components: {
+    Icon: replaceWith(SearchIcon)
```

Comment:
i'd like to establish convention that Components domain only takes tokens defined on other componets - so can we do 
Icon: cxSearchIcon.Default
(maybe use SearchIconClean as the starting compoentn fro this slot)?